certbot/certbot/_internal/storage.py

```diff
@@ -835,12 +865,18 @@ def has_pending_deployment(self):
         :rtype: bool
 
         """
+        all_versions: List[int] = []
+        for item in ALL_FOUR:
+            version = self.current_version(item)
+            if version is None:
+                raise errors.Error(f"Missing computable version for item {item}!")
```

Comment:
Although it is unlikely that a user will ever see this error message, it would be good to make it a bit easier to understand:

```suggestion
                raise errors.Error(f"{item} is required but missing for this certificate.")
```